src/SCREENS.ts

```diff
@@ -14,6 +14,11 @@ export default {
     SETTINGS: {
         PREFERENCES: 'Settings_Preferences',
         WORKSPACES: 'Settings_Workspaces',
+        SECURITY: 'Settings_Security',
+        STATUS: 'Settings_Status',
```

Comment:
Can we add documentation for when to use this pattern?

Reply:
I almost think it would be simpler to just always put screen names in the `src/SCREENS.ts` const 🤷🏼

Reply:
I think this is out-of-scope for this PR. This pattern has been around since we first started using react-navigation and we have never really documented when you should add a const to `src/SCREENS.ts` or not.

The answer has always implicitly been "whenever you need to reference the screen name somewhere else in the app".

FWIW, the screen name is defined when you create the navigator ([example](https://github.com/Expensify/App/blob/b798240d2fb4e3ae201ad2496ae9523dbd5c8eaf/src/libs/Navigation/AppNavigator/ModalStackNavigators.js#L368)). In some cases (such as this) we want to reference the screen name in some other context. In those cases we make it a constant and reference that constant when creating the navigator ([example](https://github.com/Expensify/App/blob/b798240d2fb4e3ae201ad2496ae9523dbd5c8eaf/src/libs/Navigation/AppNavigator/ModalStackNavigators.js#L382))

Reply:
Yeah, NAB- that logic makes sense here